Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -618,10 +618,12 @@ private extension ChatMessageController {
                 .deletedMessagesVisibility ?? .visibleForCurrentUser
             let shouldShowShadowedMessages = self.client.databaseContainer.viewContext.shouldShowShadowedMessages ?? false
 
+            let pageSize: Int = .messagesPageSize
```

Comment:
Note to self: This needs to come from the MessageController Query or page configuration of some sort